admin/workspace-providers/deployment/ec2.md

```diff
@@ -0,0 +1,69 @@
+---
+title: EC2
+description: Learn how to deploy a workspace provider to an EC2 cluster.
+---
+
+This article walks you through the process of deploying a workspace provider to
+an EC2 instance.
+
+## Prerequisites
+
+You must have an
+[**AWS access key ID** and **secret access key**](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys).
+
+We recommend having the [AWS CLI](https://aws.amazon.com/cli/) installed and
+configured as well.
+
+## Creating a workspace provider
+
+1. Log into Coder as a site manager, and go to **Manage** > **Workspace
+   providers**.
+
+1. In the top-right next to **Create Kubernetes Provider**, click on the **down
+   arrow** and select **Create Amazon EC2 Provider**.
+
```

Comment:
Do we need to mention that this is only available when the EC2 Feature has been enabled in the 'Feature Preview' dialog (since it's still 'Alpha')? 

![image](https://user-images.githubusercontent.com/88213859/140999352-da803698-d2d0-497e-b301-9a72e2c040a5.png)

Maybe it's already mention somewhere else.